source/clear-linux/tutorials/kubernetes/kubernetes.rst

```diff
@@ -1,7 +1,12 @@
 .. _kubernetes:
 
+<<<<<<< HEAD:source/clear-linux/tutorials/kubernetes.rst
+Run Kubernetes\*
+################
+=======
```

Comment:
Merge conflicts?!

Reply:
No.  Don't worry. I managed the merge conflicts in a separate commit.  Please review content of the commit only.

Reply:
It still doesn't make sense to me to have commits with merge commits in a PR. But if this is how you want to maintain it, so be it.